closure/goog/testing/propertyreplacer.js

```diff
@@ -116,6 +116,7 @@ goog.testing.PropertyReplacer.hasKey_ = function(obj, key) {
     //   var dummy = document.body.constructor.prototype.className
     //   [Exception... "Cannot modify properties of a WrappedNative"]
     // </pre>
+    /** @suppress {unusedLocalVariables} */
```

Comment:
Internal linter currently complains about this line, saying "@suppress is not allowed here".  Is that changing?  Can we land the linter change first?